json/cards/Neo Destiny.json

```diff
@@ -5245,6 +5245,31 @@
     "setCode": "neo4",
     "types": [
       "Darkness"
+	      ],
+	"attacks": [
+      {
+        "name": "Mountain Crush",
+        "cost": [
+          "Darkness",
+          "Fighting",
+          "Fighting"
+        ],
+        "convertedEnergyCost": 3,
+        "damage": "30",
+        "text": "Flip a coin until you get tails. For each heads, your opponent discards the top card from his or her deck."
+      },
+	  {
+        "name": "Destructive Fire",
+        "cost": [
+          "Darkness",
+          "Fire",
+          "Fire",
+		  "Fire"
```

Comment:
Very minor: The extra tabs could affect readability here. Not a huge blocker though.

Reply:
Thanks! Fixed.